lib/sles4sap_publiccloud.pm

```diff
@@ -354,8 +354,8 @@ sub stop_hana {
     $args{method} //= 'stop';
     my $timeout = bmwqemu::scale_timeout($args{timeout} // 300);
     my %commands = (
-        stop => "HDB stop",
-        kill => "HDB kill -x",
+        stop => 'HDB stop',
+        kill => 'HDB kill -x',
         crash => "echo b > /proc/sysrq-trigger &"
```

Comment:
Since you're changing the double-quotes for single quotes in the other command, please also do it for the crash command.